core/src/App.html

```diff
@@ -206,7 +206,18 @@
           navigateOk: null,
           builderCompatibilityMode: Boolean(window.builderCompatibilityMode)
         };
-
+        window.addEventListener('popstate', async e => {
+          const alert = this.get().alert;
+          if (alert && alert.ttl !== undefined) {
```

Comment:
I would use `typeof number` here, just to be save.

Reply:
I'd love to use `isInteger()` but Internet Explorer's compatibility blocks me from doing this :(